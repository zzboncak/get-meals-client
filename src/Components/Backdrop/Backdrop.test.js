import React from 'react';
import ReactDOM from 'react-dom';
import Backdrop from './Backdrop';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe(`Backdrop component`, () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<Backdrop />, div);
      ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the component as expected', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <Backdrop />
        </BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
