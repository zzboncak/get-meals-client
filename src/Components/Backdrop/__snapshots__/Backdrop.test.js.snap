// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop component renders the component as expected 1`] = `
<div
  className="backdrop"
/>
`;
